app/src/main/java/com/kodeco/android/countryinfo/flow/Flows.kt

```diff
@@ -1,10 +1,33 @@
 package com.kodeco.android.countryinfo.flow
 
+import kotlinx.coroutines.GlobalScope
+import kotlinx.coroutines.delay
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import kotlinx.coroutines.flow.update
+import kotlinx.coroutines.launch
+
 object Flows {
-    // TODO: Implement the flows here, there should be three public flows:
-    //  1. tapFlow
-    //  2. backFlow
-    //  3. counterFlow
-    //  NOTE: These public flows should be StateFlow<Int> types, you'll need to make separate
-    //  private MutableStateFlow<Int> types that can actually have their values modified.
+    private val _tapFlow = MutableStateFlow(0)
+    val tapFlow = _tapFlow.asStateFlow()
+    fun tap() {
+        _tapFlow.update { tap -> tap + 1 }
+    }
+
+    private val _backFlow = MutableStateFlow(0)
+    val backFlow = _backFlow.asStateFlow()
```

Comment:
Great use of `asStateFlow()`! Better than my suggestion to simply cast it as `StateFlow` hah